mobile_marketplace/users/api/serializers.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+User = get_user_model()
+
+
+class LoginSerializer(serializers.Serializer):
+    email = serializers.EmailField()
+    password = serializers.CharField(write_only=True)
+
+
+class RegisterSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(write_only=True)
+
+    class Meta:
+        model = User
+        fields = (
+            "email",
+            "password"
+            )
+
+    def create(self, validated_data):
+        user = User(email=validated_data["email"])
+        user.set_password(validated_data["password"])
+        user.save()
```

Comment:
Always use edit-fields to only update specific attributes instead of complete object.